docs/tests.md

```diff
@@ -0,0 +1,27 @@
+## Location of tests
+
+Take the Write API. It has:
+
+1. Unit tests in pkg/server/commands/write_test.go. These mock all the external dependencies. The tests here should test all business logic that is independent of the external dependencies.
```

Comment:
I wouldn't say it's necessarily independent of the external dependencies, but any test within these units should be targeting mocked dependencies to hit certain business logic paths.